modules/rocksdb-common/build.gradle

```diff
@@ -23,6 +23,10 @@ dependencies {
     implementation project(':ignite-core')
     implementation libs.jetbrains.annotations
     api libs.rocksdb.jni
+
+    testImplementation libs.hamcrest.core
+    testImplementation libs.hamcrest.core
```

Comment:
this line is the same as above + in pom.xml you've added 4 dependencies and here only 2 (or 3 with the duplicate)

Reply:
I added only 2 dependencies in build.gradle because other 2 dependencies seem to be already there. They are added by

```
apply from: "$rootDir/buildscripts/java-junit5.gradle"
```

probably. At least, the project compiles (including tests) via Gradle.

I removed the duplicate, thanks